Source/WebKit/WebProcess/GPU/GPUProcessConnection.cpp

```diff
@@ -166,6 +167,7 @@ void GPUProcessConnection::invalidate()
 
 void GPUProcessConnection::didClose(IPC::Connection&)
 {
+    RELEASE_LOG_ERROR(Process, "%p - GPUProcessConnection::didClose", this);
```

Comment:
Could this print the pid of the ex GPU process? That would be useful.

Reply:
I don't think we have that at the moment. This is WP code, that's privileged info. 
Connections with XPCConnection have that (WP<->UI, UI<->GPUP). 
Connections without XPCConnection, such as WP<->GPUP do not.